ui/app/templates/components/transform-show-transformation.hbs

```diff
@@ -0,0 +1,44 @@
+<div class="box is-fullwidth is-sideless is-paddingless is-marginless">
+  {{#each model.transformFieldAttrs as |attr|}}
+    {{#if (eq attr.type "object")}}
+      {{info-table-row label=(capitalize (or attr.options.label (humanize (dasherize attr.name)))) value=(stringify (get model attr.name))}}
+    {{else}}
+      {{info-table-row label=(capitalize (or attr.options.label (humanize (dasherize attr.name)))) value=(get model attr.name)}}
+    {{/if}}
+  {{/each}}
+</div>
+
+<div class="has-top-margin-xl has-bottom-margin-s">
+  <label class="title has-border-bottom-light page-header">CLI Commands</label>
+  {{!-- ARG TODO make these components? --}}
+  <div class="has-bottom-margin-s">
+    <h2 class="title is-6">Encode</h2>
+    <div class="has-bottom-margin-s">
+      <span class="helper-text has-text-grey">
+        To test the encoding capability of your transformation, use the following command. It will output an encoded_value.
+      </span>
+    </div>
+    <div class="copy-text level">
+      <code>vault write transform/encode/payments value=&lt;enter your value here&gt;</code>
+      <CopyButton class="button is-transparent level-right" @clipboardText='blahblah'
```

Comment:
The `@clipboardText` should probably be the command above, which makes me think this is a perfect use case for the handlebars {{let}} helper

Reply:
You may have to walk me through that one.  Let's touch base in the sync later today

Reply:
That's sweet!!